src/main/java/de/tum/in/www1/artemis/domain/quiz/PointCounter.java

```diff
@@ -1,28 +1,27 @@
 package de.tum.in.www1.artemis.domain.quiz;
 
-import jakarta.persistence.Column;
-import jakarta.persistence.DiscriminatorValue;
-import jakarta.persistence.Entity;
-import jakarta.persistence.ManyToOne;
+import java.io.Serializable;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
 
+import de.tum.in.www1.artemis.domain.TempIdObject;
+
 /**
  * A PointCounter.
  */
-@Entity
-@DiscriminatorValue(value = "PC")
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public class PointCounter extends QuizStatisticCounter {
-
-    @Column(name = "points")
-    private Double points;
+public class PointCounter extends TempIdObject implements Serializable {
 
-    @ManyToOne
     @JsonIgnore
     private QuizPointStatistic quizPointStatistic;
 
+    private Double points;
+
+    private Integer ratedCounter = 0;
+
+    private Integer unratedCounter = 0;
```

Comment:
Since you already initialise the counters with 0, can we use the base types and avoid nullable values?